### DISCUSSION TITLE
Question about the metadata grid in the details tab

### DISCUSSION THREAD

**Discussion by brunop00** [TT-ReBORN/Georgia-ReBORN#167] [Q&A]
How do I remove the added field from the metadata grid? Every time the number of times the song has been played is displayed, the grid reorganizes itself and the artist's logo disappears. For me, this field is unnecessary and I believe that by removing it, there will be space for the logo to remain in the details tab formatting. I added an image that illustrates the problem and which field I want to exclude

![reborn u2](https://github.com/TT-ReBORN/Georgia-ReBORN/assets/29774927/4dcaff49-63b2-43d5-b9ce-e9d8144a443f)

**Comment by TT-ReBORN** [comment]
HI @brunop00,

I guess you are using the RC-2 version. I have added a feature that automatically hides some metadata entries and only show
basic metadata when the player size is too small. This is available in the DEV version and automatically activated as default via
top menu `Options` > `Details` > `Metadata grid` > `Auto-hide full metadata on small player`.

You can download the DEV version from the master here:
https://github.com/TT-ReBORN/Georgia-ReBORN/archive/refs/heads/master.zip

**It is recommended to make a new clean portable install because I have cleaned up the Georgia-ReBORN script file structure.**
01. Please make a theme backup via `Options > Settings > Theme backup > Make backup`.
02. Now make a new portable foobar install and install the latest DEV master version from here:
https://github.com/TT-ReBORN/Georgia-ReBORN/archive/refs/heads/master.zip
03. After you have downloaded and installed the theme, copy from your old foobar your theme backup folder:
`foobar2000_old\profile\backup`
to your new:
`foobar2000\profile\backup`
04. Then copy the new custom config from: 
`foobar2000\profile\georgia-reborn\configs\georgia-reborn-custom.jsonc`
to your backup directory:
`foobar2000\profile\backup\profile\georgia-reborn\configs\`
05. Now restore your theme backup via `Options > Settings > Theme backup > Restore backup`

-TT
